### PR TITLE
GUVNOR-2367: Strange date format in Asset Search form

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2367

The issue was we were using Bootstrap 3's ```DatePicker``` whereas we should be using Uberfire Extensions ```DatePicker``` that fixes the date-formatting issue observed in the JIRA. Most of the changes in the PR are corrections to code formatting.

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml [hunk 1]
```diff
@@ -17,198 +17,198 @@
...
+                      </b:FormGroup>
+
+                      <b:FormGroup>
+                        <b:FormLabel for="dateCreated" addStyleNames="col-md-2">
+                          <ui:text from="{i18n.DateCreated}"/>
+                        </b:FormLabel>
+                        <b:Column size="MD_3">
+                          <b:Row>
+                            <b:Column size="MD_6">
+                              <ufext:DatePicker ui:field="createdAfter" placeholder="{i18n.DateAfterPlaceholder}" autoClose="true"/>
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml]: This is the change!

File: kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml [hunk 2]
```diff
@@ -17,198 +17,198 @@
...
-                <b:Heading size="H4" text="{i18n.QueryResult}"/>
-            </b:PanelHeader>
-            <b:PanelCollapse ui:field="resultAccordionCollapse" in="false">
-                <b:PanelBody>
-                    <b:Container fluid="true">
-                        <b:Row>
-                            <b:Column size="MD_12" ui:field="simplePanel">
-                                <qr:SearchResultTable/>
+                            <b:Column size="MD_6">
+                              <ufext:DatePicker ui:field="createdBefore" placeholder="{i18n.DateBeforePlaceholder}" autoClose="true"/>
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml]: This is the change!

File: kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml [hunk 3]
```diff
@@ -17,198 +17,198 @@
...
+                      </b:FormGroup>
+
+                      <b:FormGroup>
+                        <b:FormLabel for="lastModified" addStyleNames="col-md-2">
+                          <ui:text from="{i18n.LastModified}"/>
+                        </b:FormLabel>
+                        <b:Column size="MD_3">
+                          <b:Row>
+                            <b:Column size="MD_6">
+                              <ufext:DatePicker ui:field="lastModifiedAfter" placeholder="{i18n.DateAfterPlaceholder}" autoClose="true"/>
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml]: This is the change!

File: kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml [hunk 4]
```diff
@@ -17,198 +17,198 @@
...
+                            </b:Column>
+                            <b:Column size="MD_6">
+                              <ufext:DatePicker ui:field="lastModifiedBefore" placeholder="{i18n.DateBeforePlaceholder}" autoClose="true"/>
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-search-screen/kie-wb-common-search-screen-client/src/main/java/org/kie/workbench/common/screens/search/client/FindForm.ui.xml]: This is the change!

**Comment by csadilek** [comment]
+1

**Comment by csadilek** [comment]
master: 6abd3f1e5f17509fec91a13b1b0ed3a545bb246c

**Pull request closed without merging**